modules/gui/src/com/haulmont/cuba/gui/messages.properties

```diff
@@ -323,6 +323,9 @@ actions.exportSelectedTitle=Confirm export
 actions.exportSelectedCaption=Do you want to export only selected rows?
 actions.export.ALL_ROWS=All rows
 actions.export.SELECTED_ROWS=Selected rows
+actions.warningExport.title = Warning
+actions.warningExport.message = Exported table contains over 65536 records. Because XLS does not support as many rows, \
+   all entries after the 65536 row are ignored.
```

Comment:
Exported table contains more than 65536 records. Because of XLS format limitation, all records beyond 65536
 will be ignored.